packages/core-data/src/test/resolvers.js

```diff
@@ -234,6 +234,53 @@ describe( 'getEntityRecords', () => {
 			[ ENTITIES[ 1 ].kind, ENTITIES[ 1 ].name, 2 ],
 		] );
 	} );
+
+	it( 'caches permissions but does not mark entity records as resolved when using _fields', async () => {
+		const finishResolutions = jest.fn();
+		const dispatch = Object.assign( jest.fn(), {
+			receiveEntityRecords: jest.fn(),
+			receiveUserPermissions: jest.fn(),
+			__unstableAcquireStoreLock: jest.fn(),
+			__unstableReleaseStoreLock: jest.fn(),
+			finishResolutions,
+		} );
+
+		// Provide response with _links structure
+		const postTypesWithLinks = {
+			post: {
+				slug: 'post',
+				_links: {
+					self: [
+						{
+							targetHints: {
+								allow: [ 'GET', 'POST', 'PUT', 'DELETE' ],
+							},
+						},
+					],
+				},
+			},
+		};
+		triggerFetch.mockImplementation( () => postTypesWithLinks );
+
+		await getEntityRecords( 'root', 'postType', { _fields: 'slug' } )( {
```

Comment:
```suggestion
		await getEntityRecords( 'postType', 'post', { _fields: 'slug,_links' } )( {
```

Not a deal breaker, but let's try to match actual queries.

Let's also add a test and check that permission-related actions aren't dispatched when the `_links` field is missing.

Reply:
Kind of a dealbreaker to me, otherwise the test is misleading 🙃